board/genimage-raspberrypi4.cfg

```diff
@@ -0,0 +1,32 @@
+image boot.vfat {
+  vfat {
+    files = {
+      "bcm2711-rpi-4-b.dtb",
+      "rpi-firmware/cmdline.txt",
+      "rpi-firmware/config.txt",
+      "rpi-firmware/fixup.dat",
+      "rpi-firmware/start.elf",
+      "rpi-firmware/overlays",
+      "camera.txt",
+      "enable-serial-debug",
+      "zImage"
+    }
+  }
+  size = 16M
```

Comment:
Is 16M enough for the Pi4 as well?

Reply:
I haven't had any issues with it so I would presume so, this was taken from !49 and I think is just the same as the Pi zero. Perhaps worth increasing just to be safe?